### ISSUE TITLE
Pooled comps

### ISSUE THREAD

**Issue by KyleShertzer-NOAA** [mtvincen/FishGraph#20]
A few minor bugs regarding the pooled comp plots in function Comp.plots:
1. When draft mode = FALSE, the plot title should be dropped.
2. The y-axis label is not accurate if there are multiple panels and one panel is different from the others in terms of being weighted or unweighted. For example, pooled commercial length comps for scamp are weighted in the earlier block, but unweighted in the terminal block because there is only one year represented. In that case, the y-axis label says "Pooled-unweighted" but that is not true of the earlier block. 
3. Sometimes the year shown in the legend gets cut off, although this seems inconsistent. Might be better to move it from the top right slightly toward the center.

Not a bug, but nice to have:
1. Indicate Nyrs pooled in each panel, perhaps within the legend, under the first year indicating the time block.

**Comment by mtvincen** [comment]
1. I think this was fixed in the most recent version that I pushed yesterday version 2.0.2. If it is still an issue let me know.
2. With the new version I replaced the y-axis label with "proportion" when draft=FALSE. I will try to think of a way to make those labels more nuanced for when draft=TRUE
3. I think I resolved this and put a date range for each plot. If it's still cutting off the last year I'll try to figure out a solution, just let me know

I think the year range sort of does what you're requesting but only works if you don't have any gaps in the data. I'll consider adding this.

**Comment by KyleShertzer-NOAA** [comment]
Seems to work on scamp! It took a couple tries because the function arguments have changed and for whatever reason, the help utility wasn't showing me the function details after the re-install. When it finally did, I noticed one small issue: help says that max.bub has a default of 8, but the usage says 1.5.

I also got a several warnings, which I'll report to you here for your viewing pleasure:
Warning messages:
1: In par(savepar, mfrow = c(2, 1), mar = c(4, 4, 3, 1), oma = c(0,  :
  argument 1 does not name a graphical parameter
2: In par(savepar, mfrow = c(2, 1), mar = c(4, 4, 3, 1), oma = c(0,  :
  argument 1 does not name a graphical parameter
3: In par(savepar, mfrow = c(2, 1), mar = c(4, 4, 3, 1), oma = c(0,  :
  argument 1 does not name a graphical parameter
4: In qnorm(pred$Fx - U * pred$px) : NaNs produced
5: In par(savepar, mfrow = c(2, 1), mar = c(4, 4, 3, 1), oma = c(0,  :
  argument 1 does not name a graphical parameter
6: In qnorm(pred$Fx - U * pred$px) : NaNs produced
7: In par(savepar, mfrow = c(2, 1), mar = c(4, 4, 3, 1), oma = c(0,  :
  argument 1 does not name a graphical parameter
Warning message:
argument 1 does not name a graphical parameter

**Comment by mtvincen** [comment]
Glad to hear that it works. I'm not sure why I didn't get the errors for the parsave but I'm hoping that I fixed it. 
The error In qnorm(pred$Fx - U * pred$px) : NaNs produced seems to come from the calculations in the compResidual function. I'm not sure why that is occurring and I have submitted an issue on the compResidual page. It won't affect the plots other than that there will be a point that doesn't have a residual value.

I've added in a Nyrs boolean that will add this to the legend as requested
